### PR TITLE
updates for media references -2.

### PR DESCRIPTION
**Pull request by LouisBerner**

### CODE REVIEW

**Comment by LouisBerner** [comment]
If this PR needs approval, or if the commits need to be pulled into the private repo, contact @ManikaDhiman. Thanks.

**Comment by Jak-MS** [comment]
#assign: @ManikaDhiman 

@ManikaDhiman 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by ManikaDhiman** [comment]
#sign-off

File: azure-stack/hci/concepts/host-network-requirements.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ For more information about this role-based NIC qualification, please see this [l
  
 > [!NOTE]
 > The highest qualification for any adapter in our ecosystem will contain the **Management**, **Compute Premium**, and **Storage Premium** qualifications.
- 
+
 ![image](https://user-images.githubusercontent.com/12801954/188225569-bb160be0-96a2-4563-97d5-3d8efb3cb597.png)
```
Comment (Jak-MS) [inline azure-stack/hci/concepts/host-network-requirements.md:48]:
```suggestion
```
images must be hosted in the repo. @LouisBerner - I realize that your edits did not change this line, but it must be removed (and the ALT text needs to be much more descriptive).

**Comment by Jak-MS** [comment]
@ManikaDhiman - see my comment above.

FYI - @dcuomo @JasonGerend

**Comment by LouisBerner** [comment]
@alkohli 
@dcuomo 
@JasonGerend 
@ManikaDhiman 
@Jak-MS 

Revised artwork host directory and alt-text per PR reviewer suggestions from @Jak-MS.

#signoff - ready to merge and go live.

Thank you.

**Comment by LouisBerner** [comment]
@Jak-MS 

Building now without warnings or errors. Thx.

#signoff - ready to merge and go live.

Thank you.

**Review by Jak-MS — APPROVED** [review]

**Pull request merged**